package ru.sber.transport.telemechanic.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import java.time.Clock;

@Configuration
public class BeanConfig {
    
    @Bean
    public Clock clock(){
        return Clock.systemUTC();
    }
}
